pyzo/pyzokernel/debug.py

```diff
@@ -19,8 +19,9 @@ def __init__(self):
         bdb.Bdb.__init__(self)
         self._debugmode = 0  # 0: no debug,  1: postmortem,  2: full debug
         self._files_with_offset = []
-        self._original_breakpointhook = sys.breakpointhook
-        sys.breakpointhook = self.custom_breakpointhook
+        if sys.version_info >= (3, 7):
```

Comment:
Ducktyping this, because I'm a bit unsure about other interpreters (e.g. pypy) implementhing this.
```suggestion
        if hasattr(sys, "breakpointhook"):
```